src/osha/oira/client/browser/templates/preferences.pt

```diff
@@ -0,0 +1,178 @@
+                      id="patterns-label-personal-details"
+                      data-pat-collapsible="store: local; scroll-selector: self; scroll-offset: 100px"
+            >
+              <h3 class="form-separation-header collapsible-open"
+                  i18n:translate="title_personal_details"
+              >Personal details
+              </h3>
+              <div class="panel-content"><tal:widget repeat="widget view/widgets/values">
+                  <label class="">
+                ${widget/label}
+                    <input class=""
+                           name="${widget/name}"
+                           size=""
+                           type="text"
+                           value="${widget/value}"
+                    />
+                  </label><br />
+                </tal:widget>
+              </div>
+            </fieldset>
+
+            <fieldset class="form-panel pat-collapsible true section horizontal"
+                      id="label-mailings"
+                      data-pat-collapsible="store: local; scroll-selector: self; scroll-offset: 100px"
+            >
+
+              <h3 class="form-separation-header"
+                  i18n:translate="label_mailings"
+              >Mailings
+              </h3>
+              <fieldset class="pat-checklist group checkbox">
+                <legend i18n:translate="label_subscriptions">Subscriptions</legend>
+                <label class="disabled">
+                  <input checked
+                         disabled
+                         name=""
+                         type="checkbox"
+                  />
+
+                  <strong class="label-fragment-title"
+                          i18n:translate="label_functional_emails"
+                  >Functional e-mails</strong>
+                  <p class="label-fragment-description"
+                     i18n:translate="description_functional_emails"
+                  >
+							E-mails, such as password reset e-mails, that might be needed to guarantee your access to OiRA.
+                  </p>
+
+                </label>
+                <br />
+
+                <label class="">
+                  <input checked="${python:'checked' if view.has_general_subscription else None}"
+                         name="mailings.general:boolean:record"
+                         type="checkbox"
+                         value="true"
+                  />
+
+                  <strong class="label-fragment-title"
+                          i18n:translate="label_general_news"
+                  >General OiRA news</strong>
+                  <p class="label-fragment-description"
+                     i18n:translate="description_general_news"
+                  >
+							Occasional newsletters with updates about the OiRA application.
+                  </p>
+
+                </label>
+                <br />
+
+                <label class="">
```

Comment:
If the user has no session this is needed:
```
tal:condition="view/my_countries"
```
otherwise nothing will be registered in the DB